recipes/ott-jax/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "ott-jax" %}
+{% set version = "0.3.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/ott-jax-{{ version }}.tar.gz
+  sha256: adcd3c59ac9223a962b7a9244c4f53cc9857bf2461530583edb1ea8a1fc37ea3
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+  skip: true  # [py<37 or win]
```

Comment:
This one can probably be noarch if the Windows failures are only missing packages on that platform.

Reply:
I don't think this will work, I remember that in the past when submitting a recipe for a different package, having `skip: true # [win]`, the following issue was raised: 
```
    noarch packages can't have selectors. If the selectors are necessary, please remove noarch: python.
```

Reply:
Making it noarch means removing that skip. Like I said above, if the Windows failures is only a missing package, it is OK b/c the package won't be installed there but it will be ready if/when the missing dependency is available.

Reply:
`jaxlib` seems to be the cause of issues (not supported on Win) and the build fails.
I'd revert back to the selectors unless you have a suggestion as how to fix this.

Reply:
@michalk8 You misunderstand. Go ahead and remove the selectors. We can ignore the CI failure that results because it's only due to a missing dependency on Windows. In practice, noarch packages are only built (usually) on Linux, which will succeed. The package just won't be installable on Windows until (somehow) `jaxlib` packages become available. That's fine.

Keeping this noarch will greatly simplify maintenance and drastically reduce the number of builds.

Reply:
Thanks; I've been trying to figure out what's exactly wrong with `noarch` in our case (since in case for [optax](https://github.com/conda-forge/optax-feedstock/blob/main/recipe/meta.yaml), it seems to cause no problems)
Any help would be much appreciated; my guess are the pins/python version?, but doesn't seem right.

Reply:
That error is due to there being no `jaxlib` packages available on Windows.